webtester-core/src/main/java/info/novatec/testit/webtester/pageobjects/GenericList.java

```diff
@@ -0,0 +1,91 @@
+package info.novatec.testit.webtester.pageobjects;
+
+import info.novatec.testit.webtester.api.annotations.IdentifyUsing;
+import info.novatec.testit.webtester.api.annotations.Mapping;
+import info.novatec.testit.webtester.api.annotations.Mappings;
+import info.novatec.testit.webtester.api.callbacks.PageObjectCallbackWithReturnValue;
+import info.novatec.testit.webtester.api.enumerations.Method;
+
+import java.util.Collections;
+
+@Mappings({@Mapping(tag = "ol"), @Mapping(tag = "ul")})
+public class GenericList extends PageObject{
+
+    /**
+     * Returns a list of this {@link GenericList list's} {@link ListItem items}.
+     *
+     * These are only the direct children of this list. Nested lists will not be resolved!
+     *
+     * @return the list item children of this list
+     * @see GenericList
+     * @see ListItem
+     * @since 1.2
+     */
+    @IdentifyUsing(method = Method.XPATH, value = "./li")
+    private java.util.List<ListItem> listItems;
```

Comment:
Here you should import `java.utils.List`. There is no longer a name collision with the Page Object `List`.